client/store/reports/stats/selectors.js

```diff
@@ -43,7 +43,7 @@ export default {
 	 *
 	 * @param  {Object} state     Current state
 	 * @param  {String} endpoint  Stats endpoint
-	 * @param  {Object} query     Report query paremters
+	 * @param  {Object} query     Report query paremeters
```

Comment:
good catch, should be `parameters`

Reply:
OMG, introducing a new typo when I was trying to fix one. :scream: